brother_ql/backends/network.py

```diff
@@ -8,10 +8,38 @@
 from __future__ import unicode_literals
 from builtins import str
 
-import socket, os, time, select
+from pysnmp.carrier.asyncore.dispatch import AsyncoreDispatcher
+from pysnmp.carrier.asyncore.dgram import udp
+from pyasn1.codec.ber import encoder, decoder
+from pysnmp.proto import api
+from pysnmp import hlapi
 
+import socket
+from time import time
+import select
+
+import brother_ql
+
+from . import quicksnmp
 from .generic import BrotherQLBackendGeneric
 
+# Some SNMP OID's that we can use to get printer information
+brother_ql_snmp_oids = {
+    "snmp_get_ip"       : "1.3.6.1.4.1.1240.2.3.4.5.2.3.0",
+    "snmp_get_netmask"  : "1.3.6.1.4.1.1240.2.3.4.5.2.4.0",
+    "snmp_get_mac"      : "1.3.6.1.4.1.1240.2.3.4.5.2.4.0",
+    "snmp_get_location" : "1.3.6.1.2.1.1.6.0",
+    "snmp_get_model"    : "1.3.6.1.2.1.25.3.2.1.3.1",
+    "snmp_get_serial"   : "1.3.6.1.2.1.43.5.1.1.17",
+    "snmp_get_status"   : "1.3.6.1.4.1.2435.3.3.9.1.6.1.0"
```

Comment:
Question: Wouldn't enums or a class be a safer solution here (thinking about misstyped references)

Reply:
Thats a good idea, will change the code accordingly